x-team/auto-ui
# Extend body chart in note editor to 5000

**Release Type:** *Non-Breaking Feature* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2400

## Description
> f the Notes I have exceed current 1000 limit. That prevents us from editing such notes.  We should allow for 5000 characters when editing Notes for the applicant. We allow 5000 chars for portfolio items.



## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Related PRs



*N/A*

## Steps to Test or Reproduce
1- Go to admin page
2- In the note section try to update a note.
3- You should be able to see a limit of 5000 characters


## Impacted Areas in Application

- Admin >NoteFeed section > Note update



## Screenshots
![changed_words_limit](https://user-images.githubusercontent.com/5392202/49295315-e1c1b500-f48b-11e8-98ba-0e016e1f537b.png)
